django/text_extractor/templates/text_extractor/completed_documents.html

```diff
@@ -1,17 +1,61 @@
 {% load i18n %}
 
-{% for doc in ocr_docs %}
-  <tr>
-    <td>
-      <i class="bi bi-filetype-pdf me-1 h5" style="color: red"></i>
-      <a href="{% url 'text_extractor:download_document' doc.pdf.file.id user_request_id %}"
-         class="link-dark">{{ doc.pdf.file.file_name }} (PDF, {{ doc.pdf.size }})</a>
-    </td>
-    <td>
-      <i class="bi bi-filetype-txt me-1 h5" style="color: slategray"></i>
-      <a href="{% url 'text_extractor:download_document' doc.txt.file.id user_request_id %}"
-         class="link-dark">{{ doc.txt.file.file_name }} (TXT, {{ doc.txt.size }})</a>
-    </td>
-    <td style="white-space: nowrap;">{{ doc.cost }}</td>
-  </tr>
+{% for output_file in output_files %}
+  {% if output_file.pdf_file %}
+    {# This means the file is done and we can add the download links #}
+    <tr>
+      <td>
+        {# column1: Document #}
+        {{ output_file.file_name }}
+      </td>
+      <td>
+        {# column2: Status #}
+        {{ output_file.status }}
+      </td>
+      <td>
+        {# column3: Cost #}
+        {{ output_file.cost }}
+      </td>
+      <td>
+        {# column4: Results #}
+        <a href="{% url 'text_extractor:download_document' output_file.id 'pdf' %}"
+           class="link-dark me-3">
+          <i class="bi bi-filetype-pdf me-1 h5" style="color: red"></i>
+          PDF
+        </a>
+        <span class="badge bg-danger">{{ output_file.pdf_size }}</span>
+        <a href="{% url 'text_extractor:download_document' output_file.id 'txt' %}"
+           class="link-dark">
+          <i class="bi bi-filetype-txt me-1 h5" style="color: slategray"></i>
+          TXT
+        </a>
+        <span class="badge bg-secondary">{{ output_file.txt_size }}</span>
+      </td>
+    </tr>
+    <tr class="ms-3">
+      <td colspan="4" class="text-center text-warning">Please download your files before refreshing the page.</td>
```

Comment:
Added a warning message as the results will be deleted on refreshing/uploading new files